src/patternfly/components/Button/button.scss

```diff
@@ -316,6 +332,66 @@
     }
   }
 
+  &.pf-m-control {
+    position: relative;
+    border: none;
+
+    &::before {
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      content: "";
+      border: var(--pf-c-button--m-control--before--BorderWidth) solid;
+      border-color: var(--pf-c-button--m-control--before--BorderTopColor) var(--pf-c-button--m-control--before--BorderRightColor) var(--pf-c-button--m-control--before--BorderBottomColor) var(--pf-c-button--m-control--before--BorderLeftColor);
+    }
+
+    &:hover,
+    &.pf-m-hover {
+      &::before {
+        --pf-c-button--m-control--before--BorderBottomColor: var(--pf-c-button--m-control--hover--before--BorderBottomColor);
+
+        border-bottom-width: var(--pf-c-button--m-control--hover--before--BorderBottomWidth);
+      }
+    }
+
+    &:active,
+    &.pf-m-active {
+      &::before {
+        --pf-c-button--m-control--before--BorderBottomColor: var(--pf-c-button--m-control--active--before--BorderBottomColor);
+
+        border-bottom-width: var(--pf-c-button--m-control--active--before--BorderBottomWidth);
+      }
+    }
+
+    &:focus,
+    &.pf-m-focus {
+      &::before {
+        --pf-c-button--m-control--before--BorderBottomColor: var(--pf-c-button--m-control--focus--before--BorderBottomColor);
+
+        border-bottom-width: var(--pf-c-button--m-control--focus--before--BorderBottomWidth);
+      }
+    }
+
+    &.pf-m-expanded {
+      &::before {
+        --pf-c-button--m-control--before--BorderBottomColor: var(--pf-c-button--m-control--m-expanded--before--BorderBottomColor);
+
+        border-bottom-width: var(--pf-c-button--m-control--m-expanded--before--BorderBottomWidth);
+      }
+    }
+
+    &:disabled,
+    &.pf-m-disabled {
+      background-color: var(--pf-c-button--m-control--disabled--BackgroundColor);
+
+      &::before {
+        --pf-c-button--m-control--before--BorderBottomColor: var(--pf-c-button--m-control--before--BorderTopColor);
```

Comment:
This works for me, though we could also be safe and create `--pf-c-button--m-control--disabled--before--BorderBottomColor`, which allows flexbility for this to be its own var if we want to change it in the future without a breaking change. Or if you go with the approach above and create `--pf-c-button--m-control--before--BorderColor`, then you could create `--pf-c-button--m-control--disabled--before--BorderColor` and just redefine `--pf-c-button--m-control--before--BorderColor: var(--pf-c-button--m-control--disabled--before--BorderColor)` here.